website/docs/r/postgresql_flexible_server.html.markdown

```diff
@@ -10,6 +10,8 @@ description: |-
 
 Manages a PostgreSQL Flexible Server.
 
+~> **Note** As service backend would automatically change `zone` and `standby_availability_zone` after automatic failover, so TF would cause diff. For this situation, it has to add `ignore_changes` to suppress the diff.
```

Comment:
```suggestion
~> **Note** As service backend would automatically switch `zone` and `standby_availability_zone` after automatic failover, so TF would cause diff. For this situation, users need to add `ignore_changes` to suppress the diff.
```

Reply:
Updated